methods.tex

```diff
@@ -100,7 +99,7 @@ \subsection*{Governance}
 NumPy adopted an official Governance Document on October~5,
 2015\cite{NumPyProjectGovernance}.
 Project decisions are usually made by consensus of interested contributors.
-This means for many thing everyone is entrusted with veto power.
+This means for most decisions everyone is entrusted with veto power.
```

Comment:
This means that, for most decisions, everyone is...

Reply:
If this feels too strong, we could add the word "initially", or similar.